package.json

```diff
@@ -24,6 +24,7 @@
     "dep-graph": "1.1.0",
     "detect-indent": "^5.0.0",
     "elementtree": "^0.1.7",
+    "execa": "^2.0.4",
```

Comment:
Same as in another PR: Any reason to use an old version?

Reply:
@timbru31 It is probably because version 3.x was released 21 days ago and when I made these commits originally, which was cherry-picked from the draft branch, it was over 30 days ago so there was no version 3.x.

I can update the version across all repos which has this type of commit change.